addons/mail/static/src/owl/components/chatter/chatter.scss

```diff
@@ -13,7 +13,6 @@
 .o_FormRenderer_chatterContainer {
     display: flex;
     flex: 1 1 auto;
-    width: 530px;
```

Comment:
Don't see why you remove this.

Reply:
Because we don't need to fix the chatter size. It take the size left for him so that he can take full screen if its container want to. For example, when the chatter is in channel, it should take all the form sheet width but that rule wouldn't allow it

Reply:
Chatter in form sheet is a workaround to put chatter below in big screen configuration.
It does this because it has no messages to display, so it basically waste 1/3rd of a screen if it was next to it.

IIRC, this happens in rare form views, like `mail.channel` and `project.project`.
Almost every other times, we want the chatter next to form view and have a fixed width.

Reply:
The form sheet has a fixed width, so the chatter just takes the rest and it's kind of fixed. I really don't think that this rule is not needed.

In channel and project, in master, it takes all the form sheet, if we put back this rule, it won't. I don't know if there are other places where chatter is shown "full width". Do you think we need to put that back ?

Reply:
Don't get me wrong, I don't suggest just reverting the diff.
I just think that letting width _always_ being dictates with content is wrong, especially when most users see chatter on right side and in this setup expectation is having fixed width on chatter.

Reply:
Then I don't know how to make both things possible...